src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +25,27 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        StringBuilder tempString = new StringBuilder();
+        Set<String> treeSet = new TreeSet<>();
+        String result;
+        String line;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] arrS = line.replaceAll("[^a-zA-Z]", "").toLowerCase().split("");
+                treeSet.addAll(Arrays.asList(arrS));
+            }
+        } catch (IOException e) {
+            throw new FileNotFoundException("Where is the file?" + e);
```

Comment:
you will wrap any IOException to FileNotFound?We can just leave throws IOException in this method